htdocs/core/class/extrafields.class.php

```diff
@@ -1036,14 +1037,21 @@ public function fetch_name_optionals_label($elementtype, $forceload = false, $at
 		// We should not have several time this request. If we have, there is some optimization to do by calling a simple $extrafields->fetch_optionals() in top of code and not into subcode
 		$sql = "SELECT rowid, name, label, type, size, elementtype, fieldunique, fieldrequired, param, pos, alwayseditable, perms, langs, list, printable, totalizable, fielddefault, fieldcomputed, entity, enabled, help, aiprompt,";
 		$sql .= " css, cssview, csslist";
+		$parameters = ['elementtype' => $elementtype, 'forceload' => $forceload, 'attrname' => $attrname];
+		$hookmanager->executeHooks('printExtrafieldsSelect', $parameters, $this, $action);
```

Comment:
The goal of fetch_name_optionals_label is to load the definition of the system table extrafields.
There is no customization of this system table, because this table contains data to customize the business object but is not customizable itself.

Can you describe your use case why you need extrafields on extrafields table ?

Reply:
for a customer we want to add specific fields on extrafields
we made a specific join table, but we need to load thoses fields in extrafield list / view

Reply:
Sorry i don't understand what you mean with adding extrafields on extrafields.
Can you describe your use case why you need extrafields on extrafields table ?